core/src/test/scala/unit/kafka/integration/KafkaServerTestHarness.scala

```diff
@@ -58,12 +65,22 @@ trait KafkaServerTestHarness extends ZooKeeperTestHarness {
   @Before
   override def setUp() {
     super.setUp
-    if(configs.size <= 0)
+    if (configs.size <= 0)
       throw new KafkaException("Must supply at least one server config.")
     servers = configs.map(TestUtils.createServer(_)).toBuffer
     brokerList = TestUtils.getBrokerListStrFromServers(servers, securityProtocol)
     alive = new Array[Boolean](servers.length)
     Arrays.fill(alive, true)
+    // The following method does nothing by default, but
+    // if the test case requires setting up a cluster ACL,
+    // then it needs to be implemented. Check EndToEndAuthorizationTest
+    // for an example.
+    setClusterAcl match {
+      case Some(f) =>
+        f()
+        TestUtils.waitAndVerifyAcls(ClusterActionAcl, servers.head.apis.authorizer.get, clusterResource)
+      case None => // Nothing to do
+    }
```

Comment:
If we do it, then the test cases fail because of the topic creation in `IntegrationTestHarness`.